File: Helpers/MailHelper.cs
using System.Net.Mail;
using System.Web.Configuration;

namespace MyWeb.Helpers
{
    public class MailHelper
    {
        public static void SendMail(string toAddress, string subject, string body)
        {
            string smtpHost = WebConfigurationManager.AppSettings["SmtpHost"];
            int smtpPort = int.Parse(WebConfigurationManager.AppSettings["SmtpPort"]);
            string smtpUser = WebConfigurationManager.AppSettings["SmtpUser"];
            string smtpPass = WebConfigurationManager.AppSettings["SmtpPassword"];
            string fromEmail = WebConfigurationManager.AppSettings["SiteEmail"];
            string siteName = WebConfigurationManager.AppSettings["SiteName"];

            var client = new SmtpClient();
            client.Host = smtpHost;
            client.Port = smtpPort;
            client.Credentials = new System.Net.NetworkCredential(smtpUser, smtpPass);
            client.EnableSsl = true;

            var mail = new MailMessage();
            mail.From = new MailAddress(fromEmail, siteName);
            mail.To.Add(toAddress);
            mail.Subject = subject;
            mail.Body = body;
            mail.IsBodyHtml = true;

            client.Send(mail);
        }
    }
}

File: Helpers/MailHelper.vb
Imports System.Net.Mail
Imports System.Web.Configuration

Namespace My.Helpers
    Public Class MailHelper
        Public Shared Sub SendMail(toAddress As String, subject As String, body As String)
            Dim smtpHost = WebConfigurationManager.AppSettings("SmtpHost")
            Dim smtpPort = CInt(WebConfigurationManager.AppSettings("SmtpPort"))
            Dim smtpUser = WebConfigurationManager.AppSettings("SmtpUser")
            Dim smtpPass = WebConfigurationManager.AppSettings("SmtpPassword")
            Dim fromEmail = WebConfigurationManager.AppSettings("SiteEmail")
            Dim siteName = WebConfigurationManager.AppSettings("SiteName")

            Using client = New SmtpClient(smtpHost, smtpPort)
                client.Credentials = New System.Net.NetworkCredential(smtpUser, smtpPass)
                client.EnableSsl = True

                Dim mail = New MailMessage()
                mail.From = New MailAddress(fromEmail, siteName)
                mail.To.Add(toAddress)
                mail.Subject = subject
                mail.Body = body
                mail.IsBodyHtml = True

                client.Send(mail)
            End Using
        End Sub
    End Class
End Namespace
